pkg/webui/console/components/device-import-form/index.js

```diff
@@ -23,27 +23,30 @@ import FileInput from '@ttn-lw/components/file-input'
 import Checkbox from '@ttn-lw/components/checkbox'
 import SubmitBar from '@ttn-lw/components/submit-bar'
 import SubmitButton from '@ttn-lw/components/submit-button'
+import Link from '@ttn-lw/components/link'
 
 import Message from '@ttn-lw/lib/components/message'
 
 import DeviceTemplateFormatSelect from '@console/containers/device-template-format-select'
 
+import TOOLTIP_IDS from '@ttn-lw/lib/constants/tooltip-ids'
```

Comment:
```suggestion
import tooltipIds from '@ttn-lw/lib/constants/tooltip-ids'
```

Lets be consistent across the code base. Otherwise, please change this import in other places as well.